server/pages.js

```diff
@@ -35,7 +35,7 @@ const pages = routes()
   .add('collectives-iframe', '/:collectiveSlug/(collectives|widget).html')
   .add('banner-iframe', '/:collectiveSlug/banner.html')
   .add('editEvent', '/:parentCollectiveSlug/events/:eventSlug/edit/:section?')
-  .add('editCollective', '/:slug/edit/:section?')
+  .add('editCollective', '/:slug/edit/:section?/:version(v2)?')
```

Comment:
No need to add that to the route. We can just take it from the URL parameters (ie. https://opencollective.com/babel?navbarVersion=2). That way it will be available on every page using the collective navbar, and no need to touch the routes.